packages/rethinkdb/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://www.rethinkdb.com/
+TERMUX_PKG_DESCRIPTION="The open-source database for the realtime web. "
+TERMUX_PKG_LICENSE="Apache2.0"
```

Comment:
Wrong license name, must be `Apache-2.0`. Arbitrary names are not allowed neither by lint script nor by our hoster (Bintray).

Reply:
OK, I fix it